CONTRIBUTING.md

```diff
@@ -55,6 +55,7 @@ You can ask any question to the maintainer in Telegram: **[`@sitnik`](https://t.
    ```
 
 3. Run tests to be sure that everything is OK (open issue if tests don’t work in your environment):
+   &ensp; Before you run test be shure that the project located at path without any non-Latin letters in it.
```

Comment:
People do not like to read. If you put 2 long sentences together, people will read the first one and ignore the second.

It will be better to put this warning:
1. In a separated paragraph.
2. After the `pnpm test` code block to make text flow more interesting (non-boring text blocks increase readability).
3. Make it shorter.
4. Highlight `non-Latin symbols in folders` words by bold. Most people do not read, but scan. Also, it will make text flow more interesting.